src/Kernel-Tests/CompiledCodeTest.class.st

```diff
@@ -30,6 +36,12 @@ CompiledCodeTest >> testAccessesRef [
 	self assert: (writeMethod accessesRef: classVar).
 ]
 
+{ #category : #'tests - testing' }
+CompiledCodeTest >> testBlockReturnSpecial [
+
+	self deny: [ ^ 1 ] compiledBlock isReturnSpecial
```

Comment:
I am sure in a current image this should by true

Reply:
Fixed!! I was using an old image version 😅